components/TextFields/src/MDCTextInputControllerFullWidth.m

```diff
@@ -147,16 +148,23 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
       _characterCountViewMode =
           [aDecoder decodeIntegerForKey:MDCTextInputControllerFullWidthCharacterCountViewModeKey];
     }
-    _errorColor = [aDecoder decodeObjectForKey:MDCTextInputControllerFullWidthErrorColorKey];
+    _errorColor = [aDecoder decodeObjectOfClass:[UIColor class]
+                                         forKey:MDCTextInputControllerFullWidthErrorColorKey];
     _inlinePlaceholderColor =
-        [aDecoder decodeObjectForKey:MDCTextInputControllerFullWidthInlinePlaceholderColorKey];
+        [aDecoder decodeObjectOfClass:[UIColor class]
+                               forKey:MDCTextInputControllerFullWidthInlinePlaceholderColorKey];
     _inlinePlaceholderFont =
-        [aDecoder decodeObjectForKey:MDCTextInputControllerFullWidthInlinePlaceholderFontKey];
-    _textInput = [aDecoder decodeObjectForKey:MDCTextInputControllerFullWidthTextInputKey];
+        [aDecoder decodeObjectOfClass:[UIFont class]
+                               forKey:MDCTextInputControllerFullWidthInlinePlaceholderFontKey];
+    _textInput =
+        [aDecoder decodeObjectOfClass:[UIView<MDCTextInput> class]
+                               forKey:MDCTextInputControllerFullWidthInlinePlaceholderFontKey];
```

Comment:
This needs to be `MDCTextInputControllerFullWidthTextInputKey`

Reply:
Oops! Fixed these two. My eyes must have started to glaze like a ham at this point. PTAL